CRM/Extension/Mapper.php

```diff
@@ -441,7 +442,16 @@ public function getAllTags() {
    */
   public function getAllInfos() {
     foreach ($this->container->getKeys() as $key) {
-      $this->keyToInfo($key);
+      try {
+        $this->keyToInfo($key);
+      }
+      catch (CRM_Extension_Exception_ParseException $e) {
+        CRM_Core_Session::setStatus(ts('Parse error in extension: %1', [
```

Comment:
This error sounds a bit developer-y?

Reply:
Yeah - but it was already in use elsewhere - I just copied it up a level